subsys/bluetooth/host/adv.c

```diff
@@ -661,6 +661,12 @@ static int hci_set_ad_ext(struct bt_le_ext_adv *adv, uint16_t hci_op,
 		return -EAGAIN;
 	}
 
+	if (total_len_bytes > bt_dev.le.max_adv_data_len) {
+		LOG_WRN("adv or scan rsp data too large (%d > max %d)", total_len_bytes,
+			bt_dev.le.max_adv_data_len);
+		return -EDOM;
```

Comment:
We should add this error code as a `@retval` entry in the documentation for the public API functions that may call this function

Reply:
From what I can see the functions doesn't document their return code and simply say "return 0 on success or (negative) error code otherwise". If it's fine for you I'll update the documentation of those functions in a future PR.